src/vue/modules/assets-explorer/components/LogoViewer.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <img
+    v-if="url"
+    :src="url"
+    class="logo-viewer logo-viewer--image"
+  >
+  <p class="logo-viewer logo-viewer--abbr" v-else>
+    {{ asset.code | abbreviate }}
```

Comment:
What we gonna do with the widely used stuff?

- abbreviate is a filter defined in our SuperModule (gonna call the initial init of Vue in this way for now)
- also we got that form stuff that definitely gonna be pain in the ass if we try copy-paste it everywhere
- should the records share the same sharing politics like the forms and filters do? What are other components that should/shouldn’t share this politics?
- what about this `~@/scss/*` imports we do in every module?
- and the sweetest: 
  - are the module-defined vuexes gonna keep their data in the local storage? it could be very messy there eventually
  - the copy-pasted components without `scoped` styles are gonna fight with each other, since they reuse the same namespaces

My suggestion is to check how gitlab handles these points